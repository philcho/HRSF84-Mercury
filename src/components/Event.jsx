import React from 'react';

const Event = ({event}) => {
  return (
    // <a className="event container column" href={`/event/${event._id}`}>
    //   <img className="list-item-img event-img" src={event.picture}/>
    //   <div className="list-item-name event-name">{event.name}</div>
    //   <div className="list-item-bio event-bio">{event.description}</div>
    // </a>

    <a className="event container column" href="###">
      <img className="list-item-img event-img" src="http://cdn1.theodysseyonline.com/files/2015/12/04/635848643983265445-579762_christmas-puppy-pictures-tzulxzui.jpg"/>
      <div className="list-item-name event-name">Puppy Christmas</div>
      <div className="list-item-bio event-bio">There were puppies and we all played and laughed and danced!</div>
    </a>
  )
};

export default Event;
