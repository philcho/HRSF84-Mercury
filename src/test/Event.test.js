import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Event from '../components/Event';

describe('Event', () => {
  it('should render correctly with props', () => {
    const event = {
      "_id":"_fakeId",
      "name":"Some Awesome Event",
      "picture":"https://www.w3schools.com/w3css/img_fjords.jpg",
      "description":"This is a really cool event",
      "__v":0,
      "comments":[]
    };
    const output = shallow(
      <Event event={event} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

