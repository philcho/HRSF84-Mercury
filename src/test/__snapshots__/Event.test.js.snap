// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event should render correctly with props 1`] = `
<a
  className="event container column"
  href="###"
>
  <img
    className="list-item-img event-img"
    src="http://cdn1.theodysseyonline.com/files/2015/12/04/635848643983265445-579762_christmas-puppy-pictures-tzulxzui.jpg"
  />
  <div
    className="list-item-name event-name"
  >
    Puppy Christmas
  </div>
  <div
    className="list-item-bio event-bio"
  >
    There were puppies and we all played and laughed and danced!
  </div>
</a>
`;
